tests/project/integration/app/README.md

```diff
@@ -0,0 +1,3 @@
+# README
+
+This is the app/README.md file.
```

Comment:
nit: consider saying this is a simple native app used for integ tests or something like that